tests/general/LicenseCheckBearTest.py

```diff
@@ -15,7 +15,8 @@ def get_testfile_path(name):
 
 
 def load_testfile(name):
-    return open(get_testfile_path(name)).readlines()
+    with open(get_testfile_path(name)).readlines() as lines:
```

Comment:
This is not correct syntax.
Please refer https://github.com/coala/coala-bears/pull/1691/commits/5246a7f819cf98861a66f9befa50f21d999ff7dc#diff-01d6f9cb6e2d53a776a4b7ec3388b233R19 and correct it.
This is causing breaks in CI builds.